topics/climate/tutorials/ocean-variables/tutorial.md

```diff
@@ -179,7 +184,7 @@ Once you finish your work on ODV, if you want to retrieve data and/or the entire
 > You can now see bigger dots representing your data.
 > ![Image of your maps after the increase of the dots' size](../../images/ocean_var/big_dots.png)
 >
-> If you want to save it now that you already saved it once in the right folder outputs you just have to click lef on te save icon top left when it's red.
+> If you want to save it now that you already saved it once in the right folder outputs you just have to click left on the save icon top left when it's red.
```

Comment:
I think this sentence is too complicated, at least for me. Can we simplify it a bit?